7.2/Dockerfile

```diff
@@ -286,7 +286,9 @@ ENV HOME /home/docker
 ENV PATH=$PATH:$HOME/.composer/vendor/bin \
 	DRUSH_LAUNCHER_FALLBACK=/usr/local/bin/drush8
 ENV PATH=$PATH:$HOME/.terminus/vendor/bin \
-	TERMINUS_VERSION=1.8.0
+	TERMINUS_VERSION=1.8.
```

Comment:
`TERMINUS_VERSION=1.8.0`

Reply:
This actually needs to get upgrade to `1.8.1`